{{cookiecutter.project_slug}}/src/features/SplashScreen/index.js

```diff
@@ -1,55 +1,47 @@
 import React, { Component } from "react";
-import { StyleSheet, Image, ScrollView } from "react-native";
+import { StyleSheet, Image, ScrollView, TouchableOpacity } from "react-native";
 import { Text, Button } from "react-native-ui-kitten";
+import Icon from 'react-native-vector-icons/FontAwesome';
+
 import LogoIcon from "../../assets/images/backgroundLoginV1.png";
 import { installed_blueprints } from "../../config/installed_blueprints";
 import { store } from "../../redux/store";
+import { styles } from './styles'
 
 export default class App extends Component {
+
+  static navigationOptions = {
+    title: 'Installed blueprints',
+  };
+
   componentDidMount() {
     store.dispatch({ type: "TEST/ALO" });
   }
 
-  renderItem(item) {
+  renderItems() {
     const {
       navigation: { navigate }
     } = this.props;
 
     return installed_blueprints.map(item => {
       if (item.hasOwnProperty("access_route")) {
         return (
-          <Button
-            key={`${item.name}--blueprint-button`}
-            onPress={_ => navigate(item.access_route)}
-            style={styles.item}
+          <TouchableOpacity onPress={_ => navigate(item.access_route)}
+            style={styles.item} key={`${item.name}--blueprint-button`}
           >
-            {item.human_name}
-          </Button>
+            <Icon style={styles.itemLogo} name={item.icon_name} size={40} color="#F88087" />
```

Comment:
can we have default icon for a blueprint ? so that it will not a breaking change since we have a blueprint does not have icon property